etl/src/main/java/org/pih/hivmigration/etl/sql/ExamExtraMigrator.groovy

```diff
@@ -0,0 +1,64 @@
+package org.pih.hivmigration.etl.sql
+
+class ExamExtraMigrator extends SqlMigrator{
```

Comment:
Ha!  Glad I read the old comments before adding my review... I was going to suggest that this would be better named "ReturnVisitDateMigrator" but then saw that you had originally named it that and @mseaton had suggested changing it to this... :) so I'm fine keeping it like this, but I'll add my reasoning here just for reference...   I do agree that we don't want to create a migrator for every data point, but until we know how we want to group things, I thought it would make sense to say exactly what this migrator does rather than using an extrapolation (ExamExtra) that we might not use going forward.  Anyway, fine as is, but if there's another data point going forward that naturally fits with ReturnVisitDate (but isn't an "Exam Extra" data point) let's not hesitate adding it to this class and renaming this to something more appropriate.  Hope that makes sense... :)

Reply:
Agree entirely @mogoodrich .